src/renderers/widget.ts

```diff
@@ -271,6 +275,16 @@ class RenderedImage extends Widget {
     let img = document.createElement('img');
     let source = Private.getSource(options);
     img.src = `data:${options.mimeType};base64,${source}`;
+    let metadata = options.model.metadata.get(options.mimeType);
+    if (metadata) {
+      let metaJSON = metadata as JSONObject;
+      if (typeof metaJSON['height'] === "number") {
```

Comment:
Seems like that didn't work:
```
    src/renderers/widget.ts(280,15): error TS7017: Element implicitly has an 'any' type because type 'string | number | true | JSONObject | JSONArray' has no index signature.
    src/renderers/widget.ts(281,15): error TS7017: Element implicitly has an 'any' type because type 'string | number | true | JSONObject | JSONArray' has no index signature.
```

The `w` can have 5 possible types as it's type is JSONValue